starters/angular-ngrx-scss/src/app/fetch-example/loader/loader.component.scss

```
@@ -0,0 +1,30 @@
+.loading {
+  border-radius: 0.375rem;
+  border-width: 2px;
+  &__section {
+    display: flex;
+    animation: pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;
+
+    @keyframes pulse {
+      0%, 100% {
+        opacity: 1;
+      }
+      50% {
+        opacity: .5;
+      }
+    };
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+  }
+  &__wrapper {
+    display: flex;
+    margin: 0.375rem 0;
+    flex-direction: column;
+  }
+  &__item {
+    background-color: #D1D5DB;
+    border-radius: 0.375rem;
+  }
+}
```

Using flexbox for horizontal centering of this skeleton is overkill. This can be done with a block element and an automatic margin.

Also the height is set by component code at the moment which is not ideal. This can be changed to use the same font size of the text we're loading with a `:before` selector as far as the height is concerned. In regards to the width, we can use the height / max-height CSS properties set on the parent element itself.

```suggestion
:host {
  display: block;
}

.loader {
  &:before {
    content: '\00a0 ';
  }

  border-radius: 0.375rem;
  border-width: 2px;
  font-size: 1.2rem;
  margin: 1em 0;
  
  background-color: #d1d5db;
  animation: pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;

  @keyframes pulse {
    0%,
    100% {
      opacity: 1;
    }
    50% {
      opacity: 0.5;
    }
  }
}
```